cmd/fleetctl/apply.go

```diff
@@ -192,6 +204,13 @@ func applyCommand() *cli.Command {
 				log(c, "[+] applied user roles\n")
 			}
 
+			if len(specs.Teams) > 0 {
+				if err := fleetClient.ApplyTeams(specs.Teams); err != nil {
+					return errors.Wrap(err, "applying queries")
+				}
+				logf(c, "[+] applied %d teams\n", len(specs.Teams))
+			}
+
```

Comment:
I think this would be good to move above `specs.UsersRoles`, in case the user is creating a team and assigning roles within a single `apply` command.